codeclimate-govet.go

```diff
@@ -35,12 +50,14 @@ func main() {
 				pieces := strings.Split(line, ":")
 
 				if len(pieces) < 3 {
-					// engine.PrintWarning()
+					fmt.Fprintf(os.Stderr, "Error parsing the following output: %v\n", out)
+					fmt.Fprintf(os.Stderr, "\nError: %v\n", err)
```

Comment:
Where does this `err` come from? I think instead of "Error parsing ..." we can say something like "Unexpected format for the following output".